src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -100,9 +117,9 @@ public async Task StopAsync(CancellationToken cancellationToken)
             // ProcessMessageAsync invocations to cancel
             _cancellationTokenSource.Cancel();
 
-            if (_receiver != null)
+            if (Receiver != null)
```

Comment:
What will this do if the listener was never started and lazy receiver never evaluated? Will it end up creating it only to Close it?

Reply:
I believe this should fix it, but would appreciate a second look

Reply:
One minore comment on that, but otherwise OK